### DISCUSSION TITLE
Full screen option for editor

### DISCUSSION THREAD

**Discussion by RakeshPotnuru** [forem/forem#15542] [Feature Requests]
<img width="950" alt="Screenshot 2021-11-30 205212" src="https://user-images.githubusercontent.com/70439799/144075643-ff970e39-f7ce-42c4-b7af-1d2e498fbd04.png">

It's really painful to write a blog in the current editor. A fullscreen option to expand the editor would be helpful.

**Comment by Adriel85261** [comment]
As you right. Sometime we are also get panic when we are writing or managing content of [mini miltia apk new](https://minimilitiamodapk.net/mini-militia-apk-new-version/). But the best alternative thing I am doing is that I manage all the content in my MS office. And then shift it toward the blog post. Which is making our work easy. So if they provide us the full screen for us.

**Comment by blogusama** [comment]
Update your WordPress to the latest version 9.0 which was released recently. My client is also facing this error while posting in their Magazine [IQMAGS](https://iqmags.com/fifa-23-game-the-best-control-settings-camera). The New version is awesome must try it.

**Comment by sophiajhon75** [comment]
Update WordPress to the latest recently released version 9.0. My client also encounters this error when posting to her kitchenvillas. The new version is great, so give it a try.

**Comment by nencyjen** [comment]
yes facing the same [problem](https://bdsmquiz.co) kindly tell if found any solution

**Comment by Raajkumar45** [comment]
A full screen option for an editor can be helpful for users who want a larger workspace for writing. It can also help to minimize distractions and improve [freefire](https://freefirebattles.com/) focus. However, the implementation of this feature would depend on the specific editor being used and the platform it runs on. If you are using a specific platform, you can check their documentation or user forum for information on how to enable a full screen mode or if it is planned for future updates.

**Reply by inshotproapk** [reply]
As I am using Kadence theme on my site in [inshot](https://inshotapps.com/). It has a full-screen editor. which theme you are using.
